packages/reference/src/entries/WrappedEntryCard/FetchingWrappedEntryCard.tsx

```diff
@@ -21,7 +21,8 @@ export type EntryCardReferenceEditorProps = ReferenceEditorProps & {
   cardDragHandle?: React.ReactElement;
   renderCustomCard?: (
     props: CustomEntryCardProps,
-    linkActionsProps: CustomActionProps
+    linkActionsProps: CustomActionProps,
+    renderDefaultCard: (customProps?: Partial<WrappedEntryCardProps>) => React.ReactElement
```

Comment:
Could we make this `renderDefaultCard: (props: CustomEntryCardProps) => React.ReactElement` ?

The idea would be that we don't need to expose a new type `WrappedEntryCardProps` from this package and you could easily customize like `(props, _, renderCustomCard) => renderDefaultCard({...props, ...myCustomOverwriteProps})`

Reply:
Can you have a look at it now? Is that what you suggested?

Reply:
`CustomEntryCardProps` doesn't include `hasCardEditActions` which is what we want to override in Web Creator.

Reply:
@Silhoue It includes an `onEdit?: () => void;`. I believe if that's not passed the card won't feature edit functionality. Or am I mistaken?

Reply:
From what I see `onEdit` is created by `field-editors`, not passed: https://github.com/contentful/field-editors/blob/e62856c3c3ff89a82ccf9eb6af43f26695eea225/packages/reference/src/entries/WrappedEntryCard/FetchingWrappedEntryCard.tsx#L75-L88

Reply:
Isn't that just the default value which will be part of `props` in `(props, _, renderCustomCard) => renderDefaultCard({...props, ...myCustomOverwriteProps})`?

I would advise making the objective work in one of the mdx spec files first (like rendering a default card there that comes without edit functionality)

Reply:
Edit button renders when both are present https://github.com/contentful/field-editors/blob/e62856c3c3ff89a82ccf9eb6af43f26695eea225/packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx#L137

So do you suggest to pass `hasCardEditActions=true` and then overwrite the `onEdit` to `undefined` for non-page cards?

Reply:
The initial idea was to overwrite `hasCardEditActions` to `true` for page cards, which seems easier to understand.

Reply:
> So do you suggest to pass hasCardEditActions: true and then overwrite the onEdit to undefined for non-page cards?

FYI We went for the above solution as it seems to be the intention with field-editors. `hasCardEditActions` is only deciding on whether or not to show the dropdown, while `onEdit` indicates if the entity is generally openable or not and thus the pointy cursor problem you mentioned.

Reply:
Jira ticket for the pointy cursor problem mentioned above: https://contentful.atlassian.net/browse/CX-509